.cspell.json

```diff
@@ -789,6 +789,7 @@
     "packages/shared/src/types/timezones/timezones.types.ts",
     "*.riv",
     "websockets",
+    "apps/api/src/app/workflows-v2/usecases/validate-content/validate-placeholders/validate-placeholder.usecase.ts",
```

Comment:
can you add `/* cspell:disable */` on this file, i would not put here files that can be moved in the source code in the future.